Pipfile

```diff
@@ -7,10 +7,11 @@ verify_ssl = true
 coverage = "~=4.5"
 requests = "~=2.23"
 unittest-xml-reporting = "~=2.5"
+v = {editable = true,version = "*"}
```

Comment:
what is the dependency?

Reply:
good question :) I am rebuilding the pipenv and will update the Piplock. Sth must have went wrong :)

Reply:
done!